src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (num) => {
+    if (num) {
+      sum += num;
+
+      return add;
+    } else {
```

Comment:
```suggestion
    }
```
It will work without 'else'